test/services/group_links/group_link_service_test.rb

```diff
@@ -16,6 +16,12 @@ def setup
       assert_difference -> { NamespaceGroupLink.count } => 1 do
         GroupLinks::GroupLinkService.new(@user, namespace, params).execute
       end
+
+      assert_enqueued_emails 2
+      assert_enqueued_email_with GroupLinkMailer, :access_granted_user_email,
+                                 args: [Member.user_emails(group), group, namespace]
+      assert_enqueued_email_with GroupLinkMailer, :access_granted_manager_email,
+                                 args: [Member.manager_emails(group), group, namespace]
```

Comment:
All `Member.manager_emails(group)` can be updated to `Member.manager_emails(namespace)` in this test file.

Reply:
afc64834a297ec27332b860346f0d0e66c2b927b